google-cloud-language/lib/google/cloud/language.rb

```diff
@@ -54,10 +54,10 @@ module Cloud
     # ## Creating documents
     #
     # Cloud Natural Language API supports UTF-8, UTF-16, and UTF-32 encodings.
-    # (Ruby uses UTF-8 natively, which is the default sent to the API, so unless
-    # you're working with text processed in different platform, you should not
-    # need to set the encoding type.) Be aware that only English, Spanish, and
-    # Japanese language content are supported.
+    # (Ruby uses UTF-8 natively, which is the default sent to the API. To change
```

Comment:
s/`which is the default sent to the API`/`which is the default received from the API`